Formula/vlang.rb

```diff
@@ -0,0 +1,21 @@
+class Vlang < Formula
+  desc "V programming language"
+  homepage "https://vlang.io"
+  url "https://github.com/vlang/v/archive/0.1.20.tar.gz"
+  sha256 "8102b48b2c82be6be14633e76e71e215aab5221198315436f97be53e1abe1f5d"
+
+  def install
+    system "make"
```

Comment:
The upstream `Makefile` [checks out another repository with git](https://github.com/vlang/v/blob/83022a24789e340c0d425a5885742c205f1bb53f/Makefile#L11-L13). This formula will need to instead download a resource with a tagged version of the library from e.g. https://github.com/vlang/vc/releases

An issue should also be filed upstream to permit this behavior without patching the `Makefile`.

Reply:
Makefile is not used anymore and this comment does not apply anymore.